src/ontology/mondo-edit.obo

```diff
@@ -279742,7 +279741,7 @@ xref: OMIM:612417 {source="MONDO:equivalentTo"}
 xref: Orphanet:2073 {source="OMIM:612417"}
 xref: UMLS:C2676275 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:612417"}
 is_a: MONDO:0100049 {source="DC-OMIM:612417"} ! narcolepsy, susceptibility to
-relationship: predisposes_towards MONDO:0021107 {source="OMIM:612417"} ! narcolepsy
+relationship: predisposes_towards MONDO:0021107 {source="OMIM:612417"} ! hereditary narcolepsy
```

Comment:
same as above. should be "predisposes towards" narcolepsy (and not hereditary narcolepsy)